localstack/services/apigateway/helpers.py

```diff
@@ -617,25 +620,31 @@ def create_authorizer(path_payload: dict) -> Authorizer:
                     aws_apigateway_authorizer = security_config.get(
                         "x-amazon-apigateway-authorizer"
                     )
-                    return rest_api.create_authorizer(
-                        create_resource_id(),
-                        name=security_scheme_name,
-                        authorizer_type=aws_apigateway_authorizer.get("type"),
-                        provider_arns=None,
-                        auth_type=security_config.get("x-amazon-apigateway-authtype"),
-                        authorizer_uri=aws_apigateway_authorizer.get("authorizerUri"),
-                        authorizer_credentials=aws_apigateway_authorizer.get(
-                            "authorizerCredentials"
-                        ),
-                        identity_source=aws_apigateway_authorizer.get("identitySource"),
-                        identiy_validation_expression=aws_apigateway_authorizer.get(
-                            "identityValidationExpression"
-                        ),
-                        authorizer_result_ttl=aws_apigateway_authorizer.get(
-                            "authorizerResultTtlInSeconds"
+                    if authorizers.get(security_scheme_name):
+                        return authorizers.get(security_scheme_name)
+                    else:
```

Comment:
nit: `else` keyword technically not needed (could help reduce level of nesting/indentation)